frontend/src/components/Board/Settings/index.tsx

```diff
@@ -149,6 +159,40 @@ const BoardSettings = ({ isOpen, setIsOpen }: Props) => {
 		);
 	};
 
+	const configurationSettings = (
+		title: string,
+		text: string,
+		isChecked: boolean,
+		handleCheckedChanage: any,
```

Comment:
For the `handleCheckedChange` you can type this way:
```suggestion
		handleChange: (checked: boolean) => void,
```

> **Note:** you have `Chanage`, please change it for `Change` 😉